teamA/leeChanWoo/app.js

```diff
@@ -0,0 +1,93 @@
+const http = require('http');
+const express = require('express');
+const morgan = require('morgan');
+const cors = require('cors');
+const dotenv = require('dotenv').config();
+
+const { DataSource } = require('typeorm');
+
+const myDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+})
+
+myDataSource.initialize()
+    .then(() => {
+        console.log("Data Source has been initialized!")
+    })
+
+const app = express();
+
+app.use(cors());
+app.use(morgan('tiny'));
+app.use(express.json());
+
+
+// health cheak
+app.get('/ping', function (req, res, next) {
+     res.json({message: 'pong'});
+});
+
+// sign-up
+app.post('/users', async (req, res) => {
+	const { name, email, profile_image, password } = req.body
+    
+	await myDataSource.query(
+		`INSERT INTO users(
+		    name,
+		    email,
+            profile_image,
+            password
+		) VALUES (?, ?, ?, ?);
+		`,
+		[ name, email, profile_image, password ]
+	); 
```

Comment:
- SQL 문법도 가독성 좋게 만들어주는 것이 중요합니다. 회사마다 컨벤션이 다르겠지만, 위코드에서는 아래와 같은 SQL 문법의 컨벤션을 정하고 있습니다.
- 지금은 위코드에서 사용하고 있는 컨벤션을 지켜서 문법을 작성해주세요. 하지만 이는 회사마다 다르기 때문에, 정답이 아님을 반드시 인지해주세요.
- SQL 포맷팅은 다음 처럼 통일하면 좋을거 같습니다.
 ```suggestion
	await myDataSource.query(
		`INSERT INTO users(
			name,
			email,
			profile_image,
			password
		) VALUES (?, ?, ?, ?);`,
		[ name, email, profile_image, password ]
	); 
```